lib/atomic_web/components/sidebar.ex

```diff
@@ -0,0 +1,223 @@
+                      } />
+                      <%= page.title %>
+                    <% end %>
+                  </li>
+                <% end %>
+              </ul>
+            </li>
+            <%= if @is_authenticated do %>
+              <!-- Organizations listing -->
+              <%= if Enum.count(@organizations) > 0 do %>
+                <li>
+                  <div class="text-xs font-semibold leading-6 text-zinc-400"><%= gettext("Your organizations") %></div>
+                  <.live_component id="organizations_desktop" module={AtomicWeb.Components.Organizations} current_user={@current_user} current_organization={@current_organization} organizations={@organizations} />
+                </li>
+              <% end %>
+              <!-- Profile menu or "Sign in" -->
+              <li class="left-1/4 -mx-6 mt-auto -mb-2" phx-click={%JS{} |> JS.show(to: "#user-menu-button", transition: {"ease-out duration-100", "transform opacity-0 scale-95", "transform opacity-100 scale-100"})}>
+                <span class="sr-only">Open user menu</span>
+                <.dropdown orientation={:up} items={dropdown_items(@current_user)} id="user-menu-button">
+                  <:wrapper>
+                    <button class="flex select-none items-center gap-x-4 px-12 py-3 text-sm font-semibold leading-6 text-zinc-700">
+                      <.avatar
+                        name={@current_user.name}
+                        src={
+                          if @current_user.profile_picture do
+                            Uploaders.ProfilePicture.url({@current_user, @current_user.profile_picture}, :original)
+                          else
+                            nil
+                          end
+                        }
+                        size={:xs}
+                        color={:light_gray}
+                        class="!text-sm"
+                      />
+                      <span class="hidden lg:flex lg:items-center">
+                        <span class="text-sm font-semibold leading-6 text-zinc-900" aria-hidden="true"><%= @current_user.name %></span>
+                        <.icon name={:chevron_right} solid class="h-5 w-5 text-zinc-400" />
+                      </span>
+                    </button>
+                  </:wrapper>
+                </.dropdown>
+              </li>
+            <% else %>
+              <div class="mt-auto border-b border-zinc-200"></div>
+              <li class="-mx-6 flex items-center px-6 pb-3">
+                <%= live_redirect to: Routes.user_session_path(AtomicWeb.Endpoint, :new), class: "text-md font-semibold leading-6 text-zinc-900 hover:underline" do %>
+                  Sign in <span aria-hidden="true">&rarr;</span>
+                <% end %>
+              </li>
+            <% end %>
+          </ul>
+        </nav>
+      </div>
+    </div>
+    """
+  end
+
+  defp dropdown_items(current_user) do
+    [
+      %{
+        name: "Your profile",
```

Comment:
```suggestion
        name: gettext("Your profile"),
```
Since you are on this 👀